about.html

```diff
@@ -0,0 +1,162 @@
+  </header>
+
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH Work from Home"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             
+             </div>
+             <div class="mid-label">
+              <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/R.png" alt="logo"/></a>
+              </div>
+             <div class="footer-content">
+    
+              <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3>
+              <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+            
+            
+            </div>
+          </div>
+
+             <div class="middle">
+              <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+              <h3>REMOTE DEV's GLOBAL SUMMIT ~2022</h3>
+              <p>
+                Kampala, Uganda (The Pearl of AFRICA) <br />
+                Its has never been better
+              </p>
+            
+            
+            </div>
+          </footer>
+             <div class="past">
+                <h2 class="past-header">See the past Global Summits</h2> <hr>
+                <p class="desc">Take a look at the last two global summits which took place in Buenos Aires and in warsaw</p>
+               
+                <div class="past-img">
+                
+                <div class="past1"></div><div class="center">
+                  <h4>
+                    2020</h4><p>Remote Global summit 2020 warsaw</p></div>
+                  <div class="past2"></div>
+                  <div class="center1">
+                    <h4>2021</h4><p>Remote Global summit 2021 Buenos Aires</p></div>
+              </div>
+             </div>
+             <div class="partners-sect">
+                <div class="partner-header"><p>Partners</p> <hr></div>
+                <div class="partners">
+                    <img src="images/google-1-1.svg" alt="">
+                    <img src="images/coinbase.svg" alt="">
+                    <img src="images/meta-1.svg" alt="">
+                    <img src="images/microsoft.svg" alt="">
+                   
+                    <img src="images/mozilla-1.svg" alt="">
+
+                </div>
```

Comment:
- [ ] As the previous code reviewer asked please make the width and the height for the partners elements the same because in the mobile version I think they are very small 🙏🏻🙏🏻 so kindly make them bigger to match the original design as much as possible 💯💯💯